public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/EmailSharingConfiguration.tsx

```diff
@@ -53,17 +54,19 @@ const EmailList = ({
   const isLoading = isDeleteLoading || isReshareLoading;
 
   const onDeleteEmail = (recipientUid: string) => {
+    reportInteraction('grafana_dashboards_public_delete_sharing_email_clicked');
```

Comment:
I'm not sure about the event names of share, delete and reshare. Should all be named as invite, revoke and resend so there is a match between frontend events and labels?

Reply:
I would choose a name that really represents the event, despite the frontend. As already happened, frontend naming can change and shouldn't be coupled